linuxkm/linuxkm_wc_port.h

```diff
@@ -527,6 +545,7 @@
 
     #elif defined(WOLFSSL_LINUXKM_USE_SAVE_VECTOR_REGISTERS)
         #error WOLFSSL_LINUXKM_USE_SAVE_VECTOR_REGISTERS is set for an unsupported architecture.
+        #define RESTORE_VECTOR_REGISTERS() WC_RELAX_LONG_LOOP();
```

Comment:
Is this #define after #error just to give guidance on what to do for the unsupported architecture?

Reply:
nope, incomplete deletion of a fallthrough case that wound up duplicating fallthrough in types.h.  I meant to remove it completely before pushing.  fixed.